x-pack/plugins/canvas/canvas_plugin_src/functions/common/plot/series_style_to_flot.ts

```diff
@@ -46,7 +46,7 @@ export const seriesStyleToFlot = (seriesStyle: SeriesStyle) => {
     },
   };
 
-  if (stack) {
+  if (stack != null) {
```

Comment:
q: Did this fix some bug? And should the check for `color` below also be updated?

Reply:
This change just makes `0` a valid stack id number. I didn't change the falsy check for `color` because `""`/`undefined`/`null` aren't valid colors.

Reply:
That makes sense, now that I understand how flot's weird stack value works.